rulesets/data/es_lookup_by_cz.csv

```diff
@@ -0,0 +1,276 @@
+type,subtype,program,1A,1B,1C,2A,2B,2C,3A,3B,3C,4A,4B,4C,5A,5B,5C,6A,6B,6C,7,8
```

Comment:
Still need to address lookup values for things like appliances and water heaters that don't vary by climate zone.

Reply:
Rather than create a new lookup file to handle these, I would suggest incorporating it into this file (i.e., use the same value for every climate zone column). That will give us flexibility to handle, say, a program that requires a single value across all climate zones today, but changes to requiring different values for different climate zones in the future.